src/server/api/message.js

```diff
@@ -4,20 +4,14 @@ import { Message } from "../models";
 export const resolvers = {
   Message: {
     ...mapFieldsToModel(
-      [
-        "text",
-        "userNumber",
-        "contactNumber",
-        "createdAt",
-        "isFromContact",
-        "userId"
-      ],
+      ["text", "userNumber", "contactNumber", "createdAt", "isFromContact"],
       Message
     ),
     media: msg =>
       // Sometimes it's array, sometimes string. Maybe db vs. cache?
       typeof msg.media === "string" ? JSON.parse(msg.media) : msg.media || [],
     // cached messages don't have message.id -- why bother
-    id: msg => msg.id || `fake${Math.random()}`
+    id: msg => msg.id || `fake${Math.random()}`,
+    userId: msg => msg.user_id || null
```

Comment:
this was a bug I encountered while testing, where if userId was undefined the message view failed to load.